user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -27,3 +29,79 @@ Please note the following regarding suggestion events:
   - Prior to DataMiner 10.2.11/10.3.0, suggestion events are created for all anomalous behavioral changes that do not have alarm monitoring enabled.
+
+## Providing feedback on suggestion events
+
+> [!IMPORTANT]
+> The anomaly feedback feature is in preview until DataMiner 10.4.11/10.5.0. If you use the preview version of the feature, its functionality may be different from what is described below. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
+
+The anomaly feedback feature is fully available from DataMiner 10.4.11/10.5.0 onwards<!--RN 39945-->. Prior to this, it is available in soft launch from DataMiner 10.4.4 onwards, if the soft-launch option [*AnomalyFeedback*](xref:Overview_of_Soft_Launch_Options#anomalyfeedback) is enabled<!--RN 38980 + 39944-->.
+
+![Anomaly feedback feature](~/user-guide/images/Anomaly_Feedback.png)<br>*Alarm Console in DataMiner 10.4.11*
+
+Whenever behavioral anomaly detection triggers a suggestion event, you can provide feedback (positive or negative) on it in the Alarm Console. Regularly providing feedback helps anomaly detection learn from it on a system-wide level, with input from all users. This allows more accurate detection of which change points to mark as anomalies across the system<!--RN 39623-->. Feedback applies to the same parameter instance, other instances of the parameter, and other elements using the same protocol.
+
+Feedback can be provided via the ![Feedback](~/user-guide/images/Feedback_Column.png) column, which is visible by default in the *Anomalies* and *Suggestion events* tabs of the Alarm Console. To make the column visible in other tabs that list either alarms or suggestion events, right-click the column headers and select *Add or remove columns* > *Actions* > *Feedback*<!--RN 39640-->.
+
+> [!TIP]
+> See also: [*Improving anomaly detection using feedback* tutorial](xref:Anomaly_Feedback_Tutorial)
+
+### Prerequisites
+
+- Behavioral anomaly detection is enabled, and all its [prerequisites](xref:Advanced_analytics_trending) are met.
+
+- [Storage as a Service (STaaS)](xref:STaaS) or an [indexing database](xref:Supported_system_data_storage_architectures).
+
+- The [time to live](xref:Specifying_TTL_overrides) for trending is at least 1 day for real-time trending and 1 month for minute records.
+
+### Providing feedback
+
+<!--RN 39640-->
+
+To provide feedback:
+
+1. In the Alarm Console, open the *Anomalies* or *Suggestion events* tab. See [Navigating the light bulb menu](xref:Light_Bulb_Feature#navigating-the-light-bulb-menu) and [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+
+1. Verify that the ![Feedback](~/user-guide/images/Feedback_Column.png) column is visible. If it is not:
+
+   - Close the *Anomalies* or *Suggestion events* tab and reopen it. However, make sure not to use the *Reopen closed alarm lists* option. See [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
```

Comment:
It's nice to add this, as multiple people might run into this issue when first using it, but it is probably good to add when this might be necessary and why: it's only necessary if you recently upgraded from a version before feedback to a version with feedback, and you still had the tab open from before you upgraded. This is because Cube saves the opened tabs, with the columns in them and hence it will not add the new column to existing tabs.

Reply:
```suggestion
   - Close the *Anomalies* or *Suggestion events* tab and reopen it. However, make sure not to use the *Reopen closed alarm lists* option. See [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
   
    This step might be necessary if you have recently upgraded from a version of DataMiner that did not include the Feedback column to one that does (DataMiner 10.4.11/10.5.0 or higher). Since Cube saves your open tabs along with their column layout, the new column will not automatically appear in tabs that were open before the upgrade.
```